json-endpoints/class.wpcom-json-api-edit-media-v1-2-endpoint.php

```diff
@@ -0,0 +1,175 @@
+<?php
+/*
+ * WARNING: This file is distributed verbatim in Jetpack.
+ * There should be nothing WordPress.com specific in this file.
+ *
+ * @hide-in-jetpack
```

Comment:
You can remove this block -- it should only be added to wpcom as it tells our syncing script to omit it when building out the commits.